tests/dsl/test_partial.py

```diff
@@ -77,24 +77,35 @@ def test_partial_with_whitespace():
     partial = Partial[SamplePartial]
     final_model = None
     # Handle any leading whitespace from the model
-    for model in partial.model_from_chunks(["\n", "\t", " ", '{"b": {"b": 1}}']):
-        final_model = model
+    expected_model_dicts = [
+        {"a": None, "b": {}},
+        {"a": None, "b": {}},
+        {"a": None, "b": {}},
+        {"a": None, "b": {"b": 1}}
+    ]
 
-    assert final_model.model_dump() == {"a": None, "b": {"b": 1}}
+    for model, expected_dict in zip(partial.model_from_chunks(['\n', '\t', ' ', '{"b": {"b": 1}}']), expected_model_dicts):
+        assert model.model_dump() == expected_dict
 
 @pytest.mark.asyncio
 async def test_async_partial_with_whitespace():
```

Comment:
Sorry @mrdkucher for some reason I can't commit to this branch anymore.

Could you help make these edits to the branch so that the asyncio test works nicely? 

```
@pytest.mark.asyncio
async def test_async_partial_with_whitespace():
    partial = Partial[SamplePartial]
    # Handle any leading whitespace from the model
    
    expected_model_dicts = [
        {"a": None, "b": {}},
        {"a": None, "b": {}},
        {"a": None, "b": {}},
        {"a": None, "b": {"b": 1}}
    ]

    async def async_generator():
        for chunk in ["\n", "\t", " ", '{"b": {"b": 1}}']:
            yield chunk

    models = [model async for model in partial.model_from_chunks_async(async_generator())]
    for model, expected_dict in zip(models, expected_model_dicts):
        assert model.model_dump() == expected_dict
```

Reply:
I rebased and opened a new PR here, since you already fixed the tests: #1007

Reply:
Aside, are there installation instructions for contributors, along with fixing tests? It would be good to add that as a section in the contributing README

Reply:
Have merged your other PR in the meantime @mrdkucher , thanks for contributing to the repo! We have these contribution guidelines ( https://python.useinstructor.com/contributing/#contributors ) but I think we can probably make them better too.

There are some dependencies that should really be bumped